### PR TITLE
git-merge-base: add page

### PR DESCRIPTION
**Pull request by Lodifice**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 2.38.1**

### CODE REVIEW

File: pages/common/git-merge-base.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# git merge-base
+
+> Find a common ancestor of two commits.
+> More information: <https://git-scm.com/docs/git-merge-base>.
+
+- Output a best common ancestor of two commits:
```
Comment (navarroaxel) [inline pages/common/git-merge-base.md:6]:
```suggestion
- Print the best common ancestor of two commits:
```

File: pages/common/git-merge-base.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`git merge-base {{commit_1}} {{commit_2}}`
```
Comment (navarroaxel) [inline pages/common/git-merge-base.md:8]:
```suggestion
`git merge-base {{commit1}} {{commit2}}`
```

File: pages/common/git-merge-base.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Output all best common ancestors of two commits:
```
Comment (navarroaxel) [inline pages/common/git-merge-base.md:10]: `best` can only be 1.
```suggestion
- Print all merge bases for two commits:
```

File: pages/common/git-merge-base.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`git merge-base --all {{commit_1}} {{commit_2}}`
```
Comment (navarroaxel) [inline pages/common/git-merge-base.md:12]:
```suggestion
`git merge-base --all {{commit1}} {{commit2}}`
```

File: pages/common/git-merge-base.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Return 0 if the first commit is an ancestor of the second commit, else 1:
```
Comment (navarroaxel) [inline pages/common/git-merge-base.md:14]: We use `check` for validations. Perhaps this better
```suggestion
- Check if a commit is an ancestor of a specific commit:
```

File: pages/common/git-merge-base.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+`git merge-base --is-ancestor {{commit_1}} {{commit_2}}`
```
Comment (navarroaxel) [inline pages/common/git-merge-base.md:16]: Perhaps this better
```suggestion
`git merge-base --is-ancestor {{ancestor_commit}} {{commit}}`
```

File: pages/common/git-merge-base.md (hunk 2 above)
Comment (Lodifice) [reply pages/common/git-merge-base.md:8]: The page for git-range-diff also uses underscores, so I added these for consistency. Should I remove them here and also change the other page (in a separate PR maybe)?

File: pages/common/git-merge-base.md (hunk 3 above)
Comment (Lodifice) [reply pages/common/git-merge-base.md:10]: In the way git weighs merge bases, there actually can be two "best" ones, which means two of the same quality. But I agree that the wording is confusing, so maybe "all suitable" should be used?

File: pages/common/git-merge-base.md (hunk 2 above)
Comment (navarroaxel) [reply pages/common/git-merge-base.md:8]: Leave as-is.

Comment (navarroaxel) [reply pages/common/git-merge-base.md:8]: There is no rule, but I think we don't use `_` in most cases.

File: pages/common/git-merge-base.md (hunk 3 above)
Comment (navarroaxel) [reply pages/common/git-merge-base.md:10]: Ok

File: pages/common/git-merge-base.md (hunk 4 above)
Comment (CleanMachine1) [reply pages/common/git-merge-base.md:12]: Why not commit_hash instead

Comment (Lodifice) [reply pages/common/git-merge-base.md:12]: I think the git subcommand pages are not consistent in this matter, but the majority uses just commit.

Comment (EmilySeville7cfg) [reply pages/common/git-merge-base.md:12]: > I think the git subcommand pages are not consistent in this matter, but the majority uses just commit.

It means that consistency should be fixed. For instance we can write script to do that like in #9578 PR.

**Review by EmilySeville7cfg — APPROVED** [review]
No unmentioned issues found.
Thanks for a contribution! ❤️

File: pages/common/git-merge-base.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/common/git-merge-base.md:8]: > The page for git-range-diff also uses underscores, so I added these for consistency. Should I remove them here and also change the other page (in a separate PR maybe)?

All pages should be consistent, and not just some of them in this case. Numbering it's what can be automatically fixed via regex substitutions via Bash script.

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**